.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+NOTE: please submit only bug reports here, any new questions or feature requests should be submitted in Discussions:
+https://github.com/jpudysz/react-native-unistyles/discussions
+ -->
+
+## Description
+
+<!--
+A clear and concise description of what is the bug.
+-->
+
+### Related to
+
+- [ ] Components
```

Comment:
Maybe Unistyles core? We have no components 😅

Reply:
lol, just updated it